gradle.properties

```diff
@@ -7,6 +7,6 @@ yarn_mappings           = 4
 loader_version          = 0.10.6+build.214
 
 #Mod properties
-mod_version             = 1.1.5
+mod_version             = 1.2
```

Comment:
1.2 **.0**, please :)

Or even better, leave it to me ^^

Reply:
I change this in this pr now. And will leave it up to you in the future